### PR TITLE
update all examples to linkerd 1.1.1

### PR DESCRIPTION
**Pull request by siggy**

### CODE REVIEW

File: plugins/build.sbt [hunk 1]
```diff
@@ -6,7 +6,7 @@ def finagle(mod: String) =
   "com.twitter" %% s"finagle-$mod" % "6.44.0"
 
 def linkerd(mod: String) =
-  "io.buoyant" %% s"linkerd-$mod" % "1.0.2"
+  "io.buoyant" %% s"linkerd-$mod" % "1.1.1"
```
Comment (klingerf) [inline plugins/build.sbt]: We also upgraded finagle / util as part of this release, so you should update those versions as well. Actually, I'm going to do the same for the linkerd-zipkin repo, if you want to hold off until I put that up for review. Should be pretty straightforward.

Comment (klingerf) [inline plugins/build.sbt]: Here it is: https://github.com/linkerd/linkerd-zipkin/pull/11

**Comment by fjunqueira** [comment]
Hello!

I just started trying out linkerd and was having trouble getting the plugins examples to run. I could build them just fine, but got a runtime error when adding the jar to the plugins folder.

After seeing this pull request i decided to give it another go, figuring the error must had been related to the examples being made to work with a previous linkerd version. But when i try to build the examples now, i get the following error:

```
[info] Loading project definition from /home/fabio/workspace/linkerd-examples/plugins/project
[info] Set current project to plugins (in build file:/home/fabio/workspace/linkerd-examples/plugins/)
[info] Updating {file:/home/fabio/workspace/linkerd-examples/plugins/}headerClassifier...
[info] Resolving io.buoyant#finagle-h2_2.12;1.1.1 ...
[warn] 	module not found: io.buoyant#finagle-h2_2.12;1.1.1
[warn] ==== local: tried
[warn]   /home/fabio/.ivy2/local/io.buoyant/finagle-h2_2.12/1.1.1/ivys/ivy.xml
[warn] ==== jcenter: tried
[warn]   https://jcenter.bintray.com/io/buoyant/finagle-h2_2.12/1.1.1/finagle-h2_2.12-1.1.1.pom
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/io/buoyant/finagle-h2_2.12/1.1.1/finagle-h2_2.12-1.1.1.pom
[warn] ==== twitter: tried
[warn]   https://maven.twttr.com/io/buoyant/finagle-h2_2.12/1.1.1/finagle-h2_2.12-1.1.1.pom
[warn] ==== local-m2: tried
[warn]   file:/home/fabio/.m2/repository/io/buoyant/finagle-h2_2.12/1.1.1/finagle-h2_2.12-1.1.1.pom
[info] Resolving jline#jline;2.14.1 ...
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: io.buoyant#finagle-h2_2.12;1.1.1: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		io.buoyant:finagle-h2_2.12:1.1.1
[warn] 		  +- io.buoyant:grpc-runtime_2.12:1.1.1
[warn] 		  +- io.buoyant:istio_2.12:1.1.1
[warn] 		  +- io.buoyant:k8s_2.12:1.1.1
[warn] 		  +- io.buoyant:linkerd-protocol-http_2.12:1.1.1 (/home/fabio/workspace/linkerd-examples/plugins/build.sbt#L21)
[warn] 		  +- io.buoyant:header-classifier_2.12:0.1-SNAPSHOT
sbt.ResolveException: unresolved dependency: io.buoyant#finagle-h2_2.12;1.1.1: not found
	at sbt.IvyActions$.sbt$IvyActions$$resolve(IvyActions.scala:294)
	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:191)
	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:168)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:155)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:155)
	at sbt.IvySbt$$anonfun$withIvy$1.apply(Ivy.scala:132)
	at sbt.IvySbt.sbt$IvySbt$$action$1(Ivy.scala:57)
	at sbt.IvySbt$$anon$4.call(Ivy.scala:65)
	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:93)
	at xsbt.boot.Locks$GlobalLock.xsbt$boot$Locks$GlobalLock$$withChannelRetries$1(Locks.scala:78)
	at xsbt.boot.Locks$GlobalLock$$anonfun$withFileLock$1.apply(Locks.scala:97)
	at xsbt.boot.Using$.withResource(Using.scala:10)
	at xsbt.boot.Using$.apply(Using.scala:9)
	at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:58)
	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:48)
	at xsbt.boot.Locks$.apply0(Locks.scala:31)
	at xsbt.boot.Locks$.apply(Locks.scala:28)
	at sbt.IvySbt.withDefaultLogger(Ivy.scala:65)
	at sbt.IvySbt.withIvy(Ivy.scala:127)
	at sbt.IvySbt.withIvy(Ivy.scala:124)
	at sbt.IvySbt$Module.withModule(Ivy.scala:155)
	at sbt.IvyActions$.updateEither(IvyActions.scala:168)
	at sbt.Classpaths$$anonfun$sbt$Classpaths$$work$1$1.apply(Defaults.scala:1392)
	at sbt.Classpaths$$anonfun$sbt$Classpaths$$work$1$1.apply(Defaults.scala:1388)
	at sbt.Classpaths$$anonfun$doWork$1$1$$anonfun$90.apply(Defaults.scala:1422)
	at sbt.Classpaths$$anonfun$doWork$1$1$$anonfun$90.apply(Defaults.scala:1420)
	at sbt.Tracked$$anonfun$lastOutput$1.apply(Tracked.scala:37)
	at sbt.Classpaths$$anonfun$doWork$1$1.apply(Defaults.scala:1425)
	at sbt.Classpaths$$anonfun$doWork$1$1.apply(Defaults.scala:1419)
	at sbt.Tracked$$anonfun$inputChanged$1.apply(Tracked.scala:60)
	at sbt.Classpaths$.cachedUpdate(Defaults.scala:1442)
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1371)
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1325)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:235)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
[error] (headerClassifier/*:update) sbt.ResolveException: unresolved dependency: io.buoyant#finagle-h2_2.12;1.1.1: not found
[error] Total time: 9 s, completed Jul 11, 2017 3:10:51 PM

```
Is this related to this PR, or something is going on on my machine?

(if this is an issue i will create one, just making sure i'm not doing domething wrong)

**Comment by siggy** [comment]
Thanks for all the detail @fjunqueira! We'll have a look.

**Comment by klingerf** [comment]
Good catch, @fjunqueira -- looks like the 1.1.1 artifacts haven't been published yet: https://repo1.maven.org/maven2/io/buoyant/finagle-h2_2.12/

We'll get them published soon an update here when they're available.

**Comment by adleong** [comment]
That's odd.  It seems to be just the finagle-h2 project that's missing: https://repo1.maven.org/maven2/io/buoyant/linkerd-core_2.12/

**Comment by adleong** [comment]
It seems like finagle-h2 didn't get published because of this: https://github.com/linkerd/linkerd/pull/1495

**Comment by fjunqueira** [comment]
@klingerf @adleong @siggy when googling for answers i noticed there was no 1.1.1 version available, I was wondering if it was related to some missing maven repo. Turns out it wasn't. 

Anyways, I'm glad i could help :)

**Comment by klingerf** [comment]
@fjunqueira There's going to be another linkerd release later today that should have the fix for publishing the required artifact, so we'll just move this repo to the next release to fix the issue. Stay tuned, and thanks for reporting!

**Comment by klingerf** [comment]
Hey @fjunqueira, this repo has been updated to the latest release, and I've verified that the artifacts are now available. So you should be all set if you pull the latest master.

**Comment by fjunqueira** [comment]
@klingerf I'll take a look at it again soon. Thank you very much!

**Pull request merged**